### ISSUE TITLE
Featured Product & Featured Category: not enough contrast if background image is white

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-blocks#1667]
**To Reproduce**
1. Go to _Products_ > _Categories_.
2. Edit a category and upload a _Thumbnail_ image that is completely white.
3. Create a post, add a _Featured Category_ block and select that category.
4. Previsualize the post and see the background of the block is #919395 while the foreground color is #FFFFFF. That fails contrast radio tests for normal text:

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=919395

**Expected behavior**
The block should be accessible by default no matter what image is used. This could be accomplished increasing the `dimRatio` default value.

**Screenshots**
![imatge](https://user-images.githubusercontent.com/3616980/73347086-4989b080-4287-11ea-95ff-be843be00c95.png)
